tooling/FST-1028-langversion switch.md

```diff
@@ -0,0 +1,99 @@
+# F# RFC FS-1028 - Add Language version switch to fsc.exe and fsi.exe
+
+The design suggestion [FILL ME IN](https://github.com/fsharp/fslang-suggestions/issues/fill-me-in) has been marked "approved in principle".
+This RFC covers the detailed proposal for this suggestion.
+
+* [x] Approved in principle
+* [ ] [Suggestion](https://github.com/fsharp/fslang-suggestions/issues/fill-me-in)
+* [ ] Details: [under discussion](https://github.com/fsharp/fslang-design/issues/FILL-ME-IN)
+* [ ] Implementation: [In progress](https://github.com/Microsoft/visualfsharp/pull/FILL-ME-IN)
+
+
+# Summary
+[summary]: Add a new switch to fsi and fsc that allows a developer to specify the expected language version.  And to list the allowed language versions.
+
+# Motivation
+[motivation]: It's hard to add and preview new language features, because of the risk of breaking existing projects.  
```

Comment:
```suggestion
[motivation]: It's difficult to add and preview new language features, because of the risk of breaking existing projects.  Because the F# compiler doesn't support `LangVersion` today, whenever anyone updates their F# compiler they always get the latest language version, even if they didn't intend to upgrade.
```